packages/getMyPrescriptions/tests/live-spine-client.test.ts

```diff
@@ -72,7 +76,20 @@ describe("live spine client", () => {
       nhsdLoginUser: "P9:9912003071",
       errorMessage: "Request failed with status code 500",
       scenarioDescription: "spine returns an unsuccessful http status code"
-    },
+    }
+  ])(
+    "throw error when $scenarioDescription",
+    async ({httpResponseCode, spineStatusCode, nhsdLoginUser, errorMessage}) => {
+      mock.onGet("https://spine/mm/patientfacingprescriptions").reply(httpResponseCode, {statusCode: spineStatusCode})
+      const spineClient = new LiveSpineClient(logger)
+      const headers: APIGatewayProxyEventHeaders = {
+        "nhsd-nhslogin-user": nhsdLoginUser
+      }
+      await expect(spineClient.getPrescriptions(headers)).rejects.toThrow(errorMessage)
+    }
+  )
```

Comment:
We don't need any of these spine client tests here, they now exist in the spine client repo. Just move the nhs number dependant ones below to extractNHSNumber.test.ts

Reply:
Duplicated tests have been removed